modules.yml

```diff
@@ -17,6 +17,7 @@ modules:
   comp/api/authtoken: default
   comp/core/config:
     used_by_otel: true
+  comp/core/configsync: default
```

Comment:
should this have `used_by_otel: true`?

Reply:
nevermind, I see the [answer from the other comment](https://github.com/DataDog/datadog-agent/pull/34395#discussion_r1970201929)

Reply:
As of now no, this is only used in the embedded collector, not in OSS collector